test/WebSites/ValidationWebSite/Controllers/ModelMetadataTypeAttributeController.cs

```diff
@@ -38,6 +38,14 @@ public object ValidateSoftwareViewModelIncludingMetadata([FromBody] SoftwareView
             return CreateValidationDictionary();
         }
 
+        [HttpPost]
+        public object TryValidateModelWithCollectionsModel([FromBody] List<ProductViewModel> products)
```

Comment:
Talked Offline with @harshgMSFT . Not going to add a test for this case since it does not add value.